sdk/keyvault/keyvault-admin/samples/javascript/README.md

```diff
@@ -0,0 +1,70 @@
+---
+page_type: sample
+languages:
+  - javascript
+products:
+  - azure
+  - azure-key-vault
+urlFragment: keyvault-admin-javascript
+---
+
+# Azure Key Vault Administration client library samples for JavaScript
+
+These sample programs show how to use the JavaScript client libraries for Azure Key Vault Keys in some common scenarios.
+
+| **File Name** | **Description** |
+| ------------------------------- |
+| [accessControlHelloWorld.js][accessControlHelloWorld] | Lists all Role Definitions and Role Assignments, Creates a Role Assignment, then gets it and later deletes it. |
+| [backupHelloWorld.js][BackupHelloWorld] | Performs a full key backup, then a full key restore. |
+| [backupSelective.js][BackupSelective] | Performs a selective key backup, then a selective key restore. |
+
+## Prerequisites
+
+The samples are compatible with Node.js >= 8.0.0.
+
+You need [an Azure subscription][freesub] and [an Azure Key Vault][azkeyvault] to run these sample programs. To quickly create the needed Key Vault resources in Azure and to receive a connection string for them, you can deploy our sample template by clicking:
+
+[![](http://azuredeploy.net/deploybutton.png)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-sdk-for-js%2Fmaster%2Fsdk%2Fkeyvault%2Ftests-resources.json)
+
+If creating the Key Vault manually using the Azure Portal, be aware that the samples require that the soft-delete feature be enabled. Our template above will enable this feature automatically, but it is possible to enable it manually using the Azure CLI. See [this page][kvsoftdelete] for more information.
```

Comment:
It this true for these examples?

Reply:
I have a sample in which I create a key, then do a backup, then restore it, then delete the key, then flush the key. 🤔